ring/aead.rs

```diff
@@ -8,7 +8,8 @@ use test;
 fn generate_sealing_key(algorithm: &'static aead::Algorithm)
                         -> Result<aead::SealingKey, ()> {
     let mut key_bytes = vec![0u8; algorithm.key_len()];
-    try!(rand::fill_secure_random(&mut key_bytes));
+    let rnd = rand::SystemRandom::new();
```

Comment:
Thanks for doing this!

Let's follow the pattern in the `ring::rand` documentation. In particular, `generate_sealing_key` should take a `rng: &SecureRandom` parameter and the `rand::SystemRandom::new()` should happen in (or close to) the `#[bench]` function.